enhancements/update/targeted-update-edge-blocking.md

```diff
@@ -0,0 +1,180 @@
+
+## Summary
+
+This enhancement proposes a mechanism for blocking edges for the subset of clusters considered vulnerable to known issues with the stability of a particular update or target release.
+
+## Motivation
+
+When managing the [Cincinnati][cincinnati-spec] update graph [for OpenShift][cincinnati-for-openshift-design], we sometimes discover stability issues with particular release images or updates between them.
+Once an issue is discovered, we [block edges][block-edges] so we no longer recommend risky updates, or updates to risky releases.
+
+Note: as described [in the documentation][support-documentation], supported updates are still supported even if incoming edges are blocked, and Red Hat will eventually provide supported update paths from any supported release to the latest supported release in its z-stream.
+
+Incoming bugs are evaluated to determine an impact statement based on [a generic template][rhbz-1858026-impact-statement-request].
+Some bugs only impact specific platforms, or clusters with other specific features.
+For example, rhbz#1858026 [only impacted][rhbz-1858026-impact-statement] clusters with the `None` platform which were created as 4.1 clusters and subsequently updated via 4.2, 4.3, and 4.4 to reach 4.5.
+In those cases there is currently tension between wanting to protect vulnerable clusters by blocking the edge vs. wanting to avoid inconveniencing clusters which we know are not vulnerable and whose administrators may have been planning on taking the recommended update.
+
+### Goals
+
+* [Cincinnati graph-data][graph-data] maintainers will have the ability to block edges for the subset of clusters matching a particular Prometheus query.
+
+### Non-Goals
+
+* Blocking edges based on data that is not [uploaded to Telemetry][uploaded-telemetry].
+* Blocking edges for subsets of clusters when the update service is unable to reach or authenticate with an aggregating Prometheus server.
+    For example, users running a local update service will not be able to access Red Hat's internal Telemetry to determine the requesting cluster's platform, etc.
+* Specifying a particular update service implementation.
+    This enhancement floats some ideas, but the details of the chosen approach are up to each update service's maintainers.
+
+## Proposal
+
+### Enhanced graph-data schema for blocking edges
+
+[The blocked-edges schema][block-edges] will be extended with the following new properties:
+
+* `clusters` (optional, [object][json-object]), defining the subset of affected clusters.
+    If any `clusters` property matches a given cluster, the edge should be blocked for that cluster.
+    * `prometheus` (optional, [object][json-string]), with a [PromQL][] query describing affected clusters.
+        This query will be submitted to a configurable Prometheus service and should return a set of matching records with `_id` labels.
```

Comment:
We should prototype configuration here as well. This would be implemented as a policy-engine plugin, so it needs a name and configuration - Prometheus endpoint and a token